frontend/src/config.js

```diff
@@ -81,6 +81,23 @@ class Config {
       false
     );
   }
+
+  get marketplaceIntegrationEnabled() {
+    const m = this.getConfigValue('VUE_APP_MARKETPLACE_INTEGRATION', false);
+    if (m === undefined) {
+      return true;
+    } else {
+      return m;
+    }
+  }
+
+  get userGuideUrl() {
+    return this.getConfigValue('VUE_APP_USER_GUIDE_URL', false);
+  }
+
+  get githubUrl() {
+    return this.getConfigValue('VUE_APP_GITHUB_URL', false);
```

Comment:
Would the default value for these be a blank string `""` instead of boolean false?

Reply:
The second param indicates if the getConfigValue function should throw or not if the key doesn't exist:

`getConfigValue(key, throws = true) {
    let val;
    if (process.env.NODE_ENV === 'development') {
      val = process.env[key] || this.config[key];
    } else {
      val = this.config[key];
    }
    if (throws && !val) {
      throw new Error(`Configuration missing for key: ${key}`);
    }
    return val;
  }`